### PR TITLE
Kalvin branch

### PR DESCRIPTION
**Pull request by KalTang**

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -1,99 +1,107 @@
...
+          </li>
+        </ul>
+      </div>
+      <p>&copy; All Right Reserved</p>
+    </footer>
+
+    <script src="scripts/global-toggle.js"></script>
+    <script src="scripts/index.js"></script>
+  </body>
 </html>
```
Comment (Wayl123) [inline login.html:107]: We decide on using space 4 so make it into space 4 instead of space 2.

File: scss/Register/index.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+@import "register";
```
Comment (Wayl123) [inline scss/Register/index.scss:1]: Not sure if this is needed but guess it doesn't hurt.

**Review by Wayl123 — CHANGES_REQUESTED** [review]
I notice an empty file login.js, do remove that, only add it when you are using it.
As mentioned, add new breakpoint for your login page so the right side of the page doesn't cut off in the middle.

**Comment by Wayl123** [comment]
Also change space 2 to space 4.

**Review by Wayl123 — APPROVED** [review]

**Review by stevenlai4 — APPROVED** [review]
Good to merge back to main

**Pull request merged**